include/deal.II/lac/full_matrix.templates.h

```diff
@@ -1614,7 +1614,8 @@ FullMatrix<number>::left_invert(const FullMatrix<number2> &A)
   A_t.Tadd(A, 1);
   A_t.mmult(A_t_times_A, A);
   if (number(A_t_times_A.determinant()) == number(0))
-    Assert(false, ExcSingular()) else
+    Assert(false, ExcSingular());
+  else
```

Comment:
Hm, here the compiler on the serial CI complains that there is an `else` without an `if`.

Reply:
Yeah, the fixes were done according to the new version that enforces the semicolon. This is one of the shortcomings of the current implementation.